### PR TITLE
more cm not ready codes

### PR DESCRIPTION
**Pull request by dougwig**

### CODE REVIEW

File: acos_client/v21/action.py [hunk 1]
```diff
@@ -42,12 +44,13 @@ def activate_and_write(self, partition, **kwargs):
...
-            if e.msg.startswith("write memory"):
-                pass
-            else:
+        for i in range(0, 5):
+            # Request raises an exception when the "maybe error" is returned.
+            try:
+                return self._request("POST", "cli.deploy", params=None, payload=write_cmd, **kwargs)
+            except acos_errors.ACOSException as e:
+                # Catch 'might fail error'
+                if e.msg.startswith("write memory") or '2039 ' in e.msg:
```
Comment (Cedev) [inline acos_client/v21/action.py]: what's 2039?

File: acos_client/v21/action.py [hunk 2]
```diff
@@ -42,12 +44,13 @@ def activate_and_write(self, partition, **kwargs):
...
+                    time.sleep(1)
+                    continue
                 raise e
```
Comment (Cedev) [inline acos_client/v21/action.py]: What happens if, by repeated exception, the code falls through the loop?

File: acos_client/v21/action.py (hunk 1 above)
Comment (dougwig) [inline acos_client/v21/action.py]: Fantastic question.

File: acos_client/v21/action.py (hunk 2 above)
Comment (dougwig) [inline acos_client/v21/action.py]: added last_e

**Pull request merged**